src/App.js

```diff
@@ -1,38 +1,35 @@
-import React, { Component, PropTypes } from 'react';
-import { connect } from 'react-redux';
-import clone from 'lodash/clone';
-import { getData } from './actions/comment';
-import Scroll from './utils/scroll';
-import Container from './Container';
-import Item from './components/Item';
+import React from "react";
+import { connect } from "react-redux";
+import { getData } from "./actions/comment";
+import Container from "./Container";
 
 class App extends Container {
   componentWillMount() {
     const page = this.props.match.params.page;
-    this.props.dispatch(getData('news', page));
+    this.props.dispatch(getData("news", page));
   }
 
   componentWillReceiveProps(nextProps, nextState, context) {
     const newPage = nextProps.match.params.page;
     const page = this.props.match.params.page;
     if (newPage !== page) {
-      this.props.dispatch(getData('news', newPage));
+      this.props.dispatch(getData("news", newPage));
```

Comment:
I think you should run eslint, please.

Reply:
There is eslint config in CRA, I fixed all warnings generated by CRA. There is no `.eslintrc` in the project root and no npm task in `package.json`. What do you mean?

Reply:
Please:
1. I want to saw the `.eslintrc` and `precommit` in this project. 
2. Use [airbnb-eslint](https://www.npmjs.com/package/eslint-config-airbnb) is good enough.

Reply:
This PR is byproduct of my research about performance improvement. I use CRA's built in eslint configuration and prettier for formatting (which does formatting for me). You have different opinion about formatting and this is ok. What I can do is to use prettier with different configs (tell me what would you like) - I do not have time to configure `eslint` or `precommit`.

If you do not interested in those changes close PR